src/__tests__/NavBar.test.js

```diff
@@ -0,0 +1,30 @@
+import { render, screen } from '@testing-library/react';
+
+import { BrowserRouter as Router } from 'react-router-dom';
+import { Provider } from 'react-redux';
+
+import NavBar from '../components/NavBar';
+import store from '../store';
+
+describe('NavBar', () => {
+  beforeEach(() => {
+    render(
+      <Provider store={store}>
+        <Router>
+          <NavBar />
+        </Router>
+      </Provider>,
+    );
+  });
+
+  it('render NavBar page text', () => {
+    const homeButton = screen.getByText('Home');
+    expect(homeButton).toHaveTextContent('Home');
+    const myPageButton = screen.getByText('My Page');
+    expect(myPageButton).toHaveTextContent('My Page');
+    const backButton = screen.getByText('Back');
+    expect(backButton).toHaveTextContent('Back');
+    const logoutButton = screen.getByText('Logout');
+    expect(logoutButton).toHaveTextContent('Logout');
```

Comment:
```suggestion
    const homeButton = screen.getByText('Home');
    const myPageButton = screen.getByText('My Page');
    const backButton = screen.getByText('Back');
    const logoutButton = screen.getByText('Logout');

    expect(homeButton).toHaveTextContent('Home');
    expect(myPageButton).toHaveTextContent('My Page');
    expect(backButton).toHaveTextContent('Back');
    expect(logoutButton).toHaveTextContent('Logout');
```

가독성 측면에서 react-testing-library 코드들과 jest 코드들을 따로 분리하시는게 좋을 것 같습니다.

Reply:
이렇게 분리하는게 더 가독성 좋아보이긴하네요 !